Pokedex_iosTests/Pokedex_iosTests.swift

```diff
@@ -27,6 +27,24 @@ class Pokedex_iosTests: XCTestCase {
     
     func testGetPokemonsFromNetwork() {
         viewModel.getPokemons()
+        XCTAssertEqual(viewModel.pokemons.count, 4)
+    }
+    
+    func testSearchByNameSuccess() {
```

Comment:
checking on circleci (you´re free to check everytime it fails) this code you're adding shows a wrong redeclaration of these 3 functions below

Check if you're not duplicating the code that was already on develop

![imagen](https://user-images.githubusercontent.com/1596425/178397716-568f8cca-2da9-4985-a8cf-eb74c90f483a.png)

Reply:
ok, if its not yours, in this case we can fix it (just this time) as you´re the first commit with circleCI working.
Also you must have the pre-commit hooks for testing setup and also swiftlint. check on that.
for fixing it, change to:


         viewModel.searchText = "so"
         viewModel.searchPokemonsByName()

        XCTAssertEqual(viewModel.pokemons.count, 1)


the same for the other cases